sdk/python/kfp/__init__.py

```diff
@@ -16,9 +16,13 @@
 # https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)
 
-__version__ = '2.0.0b0'
+__version__ = '2.0.0-alpha.0'
 
 TYPE_CHECK = True
+
+from kfp.client import Client
```

Comment:
What do you think about adding a pylint ignore comment so that is passes a linting check?

```suggestion
from kfp.client import Client   # pylint: disable=wrong-import-position
```

Reply:
Sure, thanks for the suggestion!